### ISSUE TITLE
Convert raw-span-constants to Java constants and make it private to span-normalizer only

### ISSUE THREAD

**Issue by rish691** [hypertrace/hypertrace-ingester#71]
The entire platform consumes raw-constants. Can we do this?

**Comment by kotharironak** [comment]
@rish691 Can you describe a problem in detail?

**Comment by rish691** [comment]
This was a proposal originally made by @buchi-busireddy. 
I am assuming the intention to stop the usage of raw-span-constants in the platform and limit to the ingestion path.

**Comment by buchi-busireddy** [comment]
> This was a proposal originally made by @buchi-busireddy.
> I am assuming the intention is to stop the usage of raw-span-constants in the platform and limit to the ingestion path.

Correct. The new library that we have written should be used everywhere and these constants could become private to that library then.

**Comment by kotharironak** [comment]
@rish691 is it done?
